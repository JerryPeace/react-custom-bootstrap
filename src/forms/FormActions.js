import React from 'react';
import classNames from 'classnames';

class FormActions extends React.Component {
    render() {
        const {
            className,
            children
        } = this.props;

        return (
            <div className={classNames('tm-form-actions', className)}>
                {children}
            </div>
        )
    }
}

export default FormActions;

