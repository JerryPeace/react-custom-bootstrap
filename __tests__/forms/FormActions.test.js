import React from 'react';
import { mount } from 'enzyme';
import toJSON from'enzyme-to-json';
import FormActions from 'forms/FormActions';

describe('FormActions', () => {
    test('render form correctly', () => {
        const form = mount(
            <FormActions>
                <div>test</div>
            </FormActions>
        )
        expect(toJSON(form)).toMatchSnapshot();
    });
});
