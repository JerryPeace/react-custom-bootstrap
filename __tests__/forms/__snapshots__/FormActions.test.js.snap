// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormActions render form correctly 1`] = `
<FormActions>
  <div
    className="tm-form-actions"
  >
    <div>
      test
    </div>
  </div>
</FormActions>
`;
